### ISSUE TITLE
Build-out VA Forms Search Page and Results Page mockups 

### ISSUE THREAD

**Issue by Karak888** [department-of-veterans-affairs/va.gov-team#2340]
**Goals for VA Forms Redesign**

- redesign experience on this page to the new VA.gov look and feel.
- create new content on this page to drive traffic to top tasks and to the top digital versions of form applications.
- add analytics tracking to all links and downloads.

[mockups for 2 VA Forms pages.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/3708492/mockups.for.2.VA.Forms.pages.docx)

**Acceptance Criteria:**
- [ ] Review attached design mockups for new VA Forms Search and Results Pages 
- [ ] Mockups have been built out for review by stakeholders

**Comment by Karak888** [comment]
@ncksllvn will help with this effort and look into it.

**Comment by jenniferlee-dsva** [comment]
Uploading an updated copydeck for this work. Updated text (more recent than what's in 
the wireframes) are highlighted in yellow. 

Please use content from the copydeck rather than the design wireframes: [VA-forms-slp-conversion-paths-110519.docx - search & results views only](https://app.zenhub.com/files/133843125/169f13c4-1ecc-4322-92f8-a4fce36529ea/download)

A couple of build questions to get started with when Nick and Kelson are back - YAY Nick and Kelson:
1/ Do we build the static content in Drupal and build the dynamic content (the search and results) as a widget in Github? Or all of it in Github?

2/The forms search is driven by a very old database SQL search. This will not change for MVP 1.0. How do we connect the FE with this database? Can we build this as a prototype first in staging without that database connection?( If not, lmk and will connect you with the database owner if needed.)

There are probably other things that a FE engineer will think of which I'm not able to.

**Comment by Karak888** [comment]
@ncksllvn @kelsonic start investigating what to do with and size.  ask @jenniferlee-dsva any questions you have after investigation.

**Comment by Karak888** [comment]
will talk with Steve Walker OIT liasion on 11/14/19.

**Comment by ncksllvn** [comment]
@jenniferlee-dsva Excellent questions!

> 1/ Do we build the static content in Drupal and build the dynamic content (the search and results) as a widget in Github? Or all of it in Github?

There's a lot of content here, so I think that should live in Drupal, while the search area (the input and results area, all contained in the gray box) should be a React widget that lives in GitHub. So, similar to what we did with the Disability Rating Calculator. The blue boxes under `Top tasks for frequently downloaded VA forms` look like a new Drupal content model though, so it might be some work that involves the CMS team. 

> 2/The forms search is driven by a very old database SQL search. This will not change for MVP 1.0. How do we connect the FE with this database? Can we build this as a prototype first in staging without that database connection?( If not, lmk and will connect you with the database owner if needed.)

The VA Lighthouse team has actually done this work, so we just need to plugin to their API method. I wrote this ticket to capture that - https://github.com/department-of-veterans-affairs/va.gov-team/issues/3563.

**Comment by ncksllvn** [comment]
I took this as more of a discovery ticket, since building this page entails quite a bit. So, I broke out our tasks into a series of tickers associated with the Epic, and now I think we can close this. We can check out the Epic for what those tickets are - https://github.com/department-of-veterans-affairs/va.gov-team/issues/1632.

Also, the acceptance criteria is just about the mockups, and we definitely have those, so I think we're good to close. CC: @Karak888
